From 4cae491b9010638f8535bacb495cc49c6e50e697 Mon Sep 17 00:00:00 2001
From: toto <titi@toto.fr>
Date: Fri, 21 Feb 2020 09:50:11 +0100
Subject: [PATCH 1/2] commit readme

---
 README.md | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/README.md b/README.md
index 780dd1c..63aa7f6 100644
--- a/README.md
+++ b/README.md
@@ -1 +1,2 @@
-# m2i-jour2
\ No newline at end of file
+# m2i-jour2
+test conflit
\ No newline at end of file
-- 
2.25.0.windows.1

